M365/src/DLT365Groupsupgrade.ps1

```diff
@@ -132,9 +132,9 @@ Function Debuggroupnesting {
     foreach ($parentdg in $alldgs) {
         try {
             $Pmembers = Get-DistributionGroupMember $($parentdg.Guid.ToString()) -ErrorAction Stop
-            $CurrentDescription = "Retrieving: $parentdg members"
-            $CurrentStatus = "Success"
-            log -Function "Retrieve Distribution Group membership" -CurrentDescription $CurrentDescription -CurrentStatus $CurrentStatus
+            #$CurrentDescription = "Retrieving: $parentdg members"
```

Comment:
More commented out code.

Reply:
I did intend to remove these by commenting them and I have removed them totally now